gnomad_qc/v4/create_release/calculate_variant_statistics.py

```diff
@@ -1,19 +1,46 @@
-"""Script to get per-sample stats by variant type in v4."""
+"""
+Script to get per-sample variant counts and aggregate sample statistics.
+
+The following per-sample variant counts can be calculated:
+
+    - Total number of variants
+    - Number of variants that pass all variant qc filters
+    - Number of variants in UK Biobank capture regions
+    - Number of variants in Broad capture regions
+    - Number of rare variants (adj AF <0.1%)
+    - Number of loss-of-function variants
+    - Number of missense variants
+    - Number of synonymous variants
+    - Number of singletons
+    - Number of heterozygous variants
+    - Number of homozygous variants
+    - Number of non-reference variants
+    - Number of variants in UK Biobank and Broad capture regions
+
+The following aggregate sample stats of all of the above per-sample counts can be
+computed:
+
+        - Mean
+        - Quantiles (0.0, 0.25, 0.5, 0.75, 1.0)
+
+Aggregated statistics can also be computed by ancestry.
+"""
+# TODO: Maybe move to a folder called assessment and rename to
```

Comment:
@matren395 @KoalaQin thoughts on this TODO?

Reply:
does 'assessment' exist in older versions or other projects? 

this would be creating stats to go with the release, so I'm very okay leaving it in create_release. However, if more assessments are coming down the road, then making a new folder could make sense? I think I'm leaning against, but not super strongly

Reply:
We don't have `assesment` under `gnomad_qc`, but we do have it under `gnomad_methods` and there's the summary_stats.py there. One question though, are we publicly releasing this per_sample count table or just the final aggregate stats? I'm not sure if we're allowed to, like `meta` isn't public.

Reply:
Correct, we have not productionized release stats before, but I was using assessment because that is what we used for some release level stats that went into gnomad_methods. Nope, we can't release the HT, and the stats wont actually be fully released, we will add some of them to the stats page or to other pages of the browser. That's why I was thinking moving them to an assessment folder where we can also put the script in this [PR](https://github.com/broadinstitute/gnomad_qc/pull/501) into it.